### PR TITLE
Update Italian Translations

### PR DESCRIPTION
**Pull request by andscape-dev**
Update the translations to Italian added by @andreaippo in #533, based on discussions with the community on Feddit.it ([thread](https://beehaw.org/post/467353)).

Notable updates:
- Attempt to remove gendered nouns for users
- Standardize capitalization for UI elements
- Fix gender suffixes (ex: "Tutte" -> "Tutti" for Posts)
- Fix specific strings based on user feedback from thread (ex: "Mostra Sorgente" -> "Mostra Markdown")
- Add translations for strings added in recent commits

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -122,100 +122,162 @@
...
+    <string name="bottomBar_communityList">Vai Alla Lista Delle Comunità</string>
+    <string name="bottomBar_bookmarks">Vai Ai Miei Segnalibri</string>
+    <string name="bottomBar_profile">Vai Al Mio Profilo</string>
+    <string name="bottomBar_inbox">Vai Alla Mia Inbox</string>
+    <string name="commentOrPostHeader_deleted">Post o Commento Eliminato</string>
+    <string name="addBookmark">Aggiungi Ai Segnalibri</string>
+    <string name="removeBookmark">Rimuovi Dai Segnalibri</string>
+    <string name="commentFooter_reply">Rispondi A Questo Commento</string>
+    <string name="dialog_moreOptions">Mostra Più Opzioni</string>
+    <string name="pickImage_imagePreview">Anteprima dell\' Immagine Caricata</string>
```
Comment (andreaippo) [inline app/src/main/res/values-it/strings.xml:238]: Extra space after apostrophe

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -122,100 +122,162 @@
...
+    <string name="person_routines_blocked">%1$s Bloccati</string>
+    <string name="utils_login_first">Effettua prima il Login</string>
+    <string name="login_view_model_incorrect_login">Login Errato</string>
     <string name="login_view_model_is_not_a_lemmy_instance">%1$s non è un\'istanza Lemmy</string>
-    <string name="create_report_view_model_report_created">Report creato</string>
+    <string name="create_report_view_model_report_created">Segnalazione Creata</string>
     <string name="about_social">Social</string>
     <string name="app_bars_dot_spacer">·</string>
     <string name="comment_node_in">"in "</string>
     <string name="look_and_feel_activity_show_content_for_collapsed_comments">Mostra contenuto per commenti compressi</string>
```
Comment (andreaippo) [inline app/src/main/res/values-it/strings.xml:200]: I know I provided the initial translation, but now I'm wondering whether this alternative is preferable:
Mostra Contenuto dei Commenti Minimizzati

File: app/src/main/res/values-it/strings.xml [hunk 3]
```diff
@@ -122,100 +122,162 @@
...
-    <string name="look_and_feel_post_view_small_card">Tessera piccola</string>
+    <string name="look_and_feel_post_view_small_card">Tessera Piccola</string>
     <string name="look_and_feel_post_view_list">Lista</string>
-    <string name="look_and_feel_show_navigation_bar">Mostra barra di navigazione</string>
+    <string name="look_and_feel_show_navigation_bar">Mostra barra di navigazione </string>
     <string name="look_and_feel_use_custom_tabs">Usa schede personalizzate</string>
     <string name="settings_activity_settings">Impostazioni</string>
-    <string name="settings_activity_look_and_feel">Aspetto</string>
+    <string name="settings_activity_look_and_feel">Aspetto Interfaccia</string>
     <string name="settings_activity_account_settings">%1$s impostazioni</string>
```
Comment (andreaippo) [inline app/src/main/res/values-it/strings.xml:188]: Missing capitalization for Impostazioni

File: app/src/main/res/values-it/strings.xml [hunk 4]
```diff
@@ -122,100 +122,162 @@
...
+    <string name="account_settings_bot_account">Impostati Come Bot</string>
+    <string name="account_settings_show_bot_accounts">Mostra Bot</string>
+    <string name="account_settings_show_scores">Mostra Punteggi</string>
+    <string name="account_settings_show_notifications_for_new_posts">Invia Notifiche per Nuovi Post</string>
+    <string name="account_settings_send_notifications_to_email">Invia Notifiche via Email</string>
+    <string name="account_settings_save_settings">Salva Impostazioni</string>
+    <string name="account_settings_activity_account_settings">Impostazioni Account</string>
     <string name="look_and_feel_look_and_feel">Aspetto</string>
-    <string name="look_and_feel_font_size">Dimensione carattere: %1$s</string>
+    <string name="look_and_feel_font_size">Grandezza font: %1$s</string>
```
Comment (andreaippo) [inline app/src/main/res/values-it/strings.xml:172]: I see various changes to replace leftover yet very popular English words with their Italian counterpart (e.g. Inbox, which you changed to Posta In Arrivo, Community to Comunità, and so on).
Yet here you do the opposite, prefering Font over Carattere. We should try and be more consistent.

I need to get some inspiration from other apps, but for example Android translates this, in the system Settings, as "Dimensioni (plural) carattere". So yeah, I'd rather go back to the previous translation here, maybe turning dimensione into plural form.

File: app/src/main/res/values-it/strings.xml [hunk 5]
```diff
@@ -122,100 +122,162 @@
...
-    <string name="account_settings_activity_account_settings">Impostazioni account</string>
+    <string name="account_settings_show_avatars">Mostra Avatar</string>
+    <string name="account_settings_show_read_posts">Mostra Post Già Letti</string>
+    <string name="account_settings_bot_account">Impostati Come Bot</string>
+    <string name="account_settings_show_bot_accounts">Mostra Bot</string>
+    <string name="account_settings_show_scores">Mostra Punteggi</string>
+    <string name="account_settings_show_notifications_for_new_posts">Invia Notifiche per Nuovi Post</string>
+    <string name="account_settings_send_notifications_to_email">Invia Notifiche via Email</string>
+    <string name="account_settings_save_settings">Salva Impostazioni</string>
+    <string name="account_settings_activity_account_settings">Impostazioni Account</string>
```
Comment (andreaippo) [inline app/src/main/res/values-it/strings.xml:170]: In a previous translation  you replace account with profilo, but here it's account again. Which one?

File: app/src/main/res/values-it/strings.xml [hunk 6]
```diff
@@ -122,100 +122,162 @@
...
     <string name="account_settings_new">Nuovi</string>
-    <string name="account_settings_topday">Top giorno</string>
-    <string name="account_settings_topweek">Top settimana</string>
-    <string name="account_settings_topmonth">Top mese</string>
-    <string name="account_settings_topyear">Top anno</string>
-    <string name="account_settings_topall">Top di tutti i tempi</string>
-    <string name="account_settings_mostcomments">Più commenti</string>
-    <string name="account_settings_newcomments">Nuovi commenti</string>
-    <string name="account_settings_default_sort_type">Ordinamento predefinito</string>
+    <string name="account_settings_topday">TopGiorno</string>
```
Comment (andreaippo) [inline app/src/main/res/values-it/strings.xml:153]: These are values for a drop-down. While I know that in English there's no spacing to separe multiple words at this location, I wonder if that's a glitch, and what would be the problem with using spaces to separate words here.

File: app/src/main/res/values-it/strings.xml
Comment (andreaippo) [inline app/src/main/res/values-it/strings.xml:1]: Thanks for the great work, you caught a few slips here and there as well. 👍
In general, though, I can't say I'm a fan of capitalizing every noun and sometimes even articles?
It's not the practice of our language (as opposed to, say, German).
I would much rather prefer capitalizing the first letter of a sentence and be done with it.
I'm currently on my mobile and the first app I have at hand (Firefox) doesn't do this neither (e.g. Salva in una raccolta, Aggiungi alle scorciatoie)

File: app/src/main/res/values-it/strings.xml [hunk 7]
```diff
@@ -122,100 +122,162 @@
...
+    <string name="community_sortBy">Ordina Per</string>
+    <string name="floating_createPost">Crea Post</string>
+    <string name="community_icon">Icona dellla Comunità</string>
+    <string name="form_submit">Invia</string>
+    <string name="createPost_selectCommunity">Seleziona Comunità dalla Lista</string>
+    <string name="commentMention_link">Vai al Commento</string>
+    <string name="markRead">Segna Come Già Letto</string>
+    <string name="markUnread">Segna Come Non Letto</string>
+    <string name="comment_save">Salva Commento</string>
+    <string name="comment_unsave">De-salva Commento</string>
```
[the remark this answers is not in the document]
Comment (andscape-dev) [reply app/src/main/res/values-it/strings.xml:250]: Yeah, de-salva sucks hahah, I couldn't come up with a better opposite for "salvare". "Ignora Commento" could work, but I'm not sure it's clear to users that it unsaves a comment...

File: app/src/main/res/values-it/strings.xml (hunk 2 above)
Comment (andscape-dev) [reply app/src/main/res/values-it/strings.xml:200]: I agree, I think we can even remove "contenuto". "Mostra commenti minimizzati" I think gets the point across while being shorter.

File: app/src/main/res/values-it/strings.xml (hunk 3 above)
Comment (andscape-dev) [reply app/src/main/res/values-it/strings.xml]: Fixed

File: app/src/main/res/values-it/strings.xml (hunk 5 above)
Comment (andscape-dev) [reply app/src/main/res/values-it/strings.xml]: Fixed using Profilo everywhere

**Comment by Magneticdud** [comment]
` <string name="markdownHelper_formatStrikethrough">Formatta ComeTesto Barrato</string>` manca lo spazio

Ma perché ogni parola è maiuscola? Per esempio  `<string name="bottomBar_communityList">Vai Alla Lista Delle Comunità</string>`.

La Leggibilità Nel Caso Ogni Parola Sia Maiuscola Mi Pare Inferiore. A Meno Che Non Sia Una Scelta Stilistica Dell'App Che Si Riflette Anche Nelle Altre Lingue, In Italiano, Solo I Nomi Propri Si Scrivono In Maiuscolo

**Review by dessalines — COMMENTED** [review]
Should I hold off on merging this until everyone's satisfied? Or would everyone prefer to merge this, then do additional PRs for small fixes?

File: app/src/main/res/values-it/strings.xml (hunk 4 above)
Comment (andreaippo) [reply app/src/main/res/values-it/strings.xml]: ok, then back to Dimensione Carattere is fine by me

File: app/src/main/res/values-it/strings.xml (hunk 6 above)
Comment (andreaippo) [reply app/src/main/res/values-it/strings.xml]: Great! I wonder if you'd feel like opening a MR on the original (English) strings as well, maybe it's indeed a glitch by the dev ;)

**Comment by andreaippo** [comment]
> Should I hold off on merging this until everyone's satisfied? Or would everyone prefer to merge this, then do additional PRs for small fixes?

Hey, while I still have some remarks, I'm personally a fan of the "done is better than perfect" approach.
So if the others also agree I'd say merge it.
We can keep discussing our different POVs on this MR, and depending on the outcome open a small follow-up one.

**Comment by andreaippo** [comment]
> 

Concordo, ed è il commento che ho fatto anch'io qui:

https://github.com/dessalines/jerboa/pull/591#discussion_r1228045328

**Comment by andreaippo** [comment]
> ` <string name="markdownHelper_formatStrikethrough">Formatta ComeTesto Barrato</string>` manca lo spazio
> 
> Ma perché ogni parola è maiuscola? Per esempio `<string name="bottomBar_communityList">Vai Alla Lista Delle Comunità</string>`.
> 
> La Leggibilità Nel Caso Ogni Parola Sia Maiuscola Mi Pare Inferiore. A Meno Che Non Sia Una Scelta Stilistica Dell'App Che Si Riflette Anche Nelle Altre Lingue, In Italiano, Solo I Nomi Propri Si Scrivono In Maiuscolo

Concordo, ed è il commento che ho fatto anch'io qui:

https://github.com/dessalines/jerboa/pull/591#discussion_r1228045328

**Review by andreaippo — APPROVED** [review]

**Comment by andscape-dev** [comment]
Thanks @andreaippo for the review! I've fixed most of the issues you pointed out, let me know if we can resolve the convos. I haven't changed "de-salva" yet, can we come up with a clearer solution?

For the capitalization I don't have a strong opinion. I do prefer capitalization for UI labels, since for me they're closer to titles than body text, but that might be because I'm used to English locales. It's also used in the English version of Jerboa, even though English grammar doesn't require capitalizing nouns. However, if it's standard not to capitalize labels in the Italian locale, we can revert to that format. I would say let's keep this for another PR though.

File: app/src/main/res/values-it/strings.xml (hunk 7 above)
[the remark this answers is not in the document]
Comment (andreaippo) [reply app/src/main/res/values-it/strings.xml:250]: Oh I see maybe, this is to "star" a comment for finding it back later on?
I though it had to do with comment editing features instead, hence my proposal to go with "Ignora".
How about "Rimuovi da commenti salvati"?

**Comment by andreaippo** [comment]
> Thanks @andreaippo for the review! I've fixed most of the issues you pointed out, let me know if we can resolve the convos. I haven't changed "de-salva" yet, can we come up with a clearer solution?
> 
> For the capitalization I don't have a strong opinion. I do prefer capitalization for UI labels, since for me they're closer to titles than body text, but that might be because I'm used to English locales. It's also used in the English version of Jerboa, even though English grammar doesn't require capitalizing nouns. However, if it's standard not to capitalize labels in the Italian locale, we can revert to that format. I would say let's keep this for another PR though.

Indeed let's try to address the ugly de-salva first LOL
As for capitalization, as @Magneticdud and I said in other comments, it looks clunky and out of place in Italian apps. Any Android app I'm opening now (Gmail, Pixel dialer) further backs this position. But I agree about handling this in another MR ;)

**Comment by andscape-dev** [comment]
@andreaippo How about "scarta" instead of de-salva? Gives the idea of something you have and want to throw away...

**Comment by andscape-dev** [comment]
@dessalines I think we're ready

**Review by dessalines — APPROVED** [review]
Mmmk great!

**Pull request merged**